framework/arcane-framework/src/main/scala/services/lakehouse/IcebergS3CatalogWriter.scala

```diff
@@ -114,6 +127,6 @@ object IcebergS3CatalogWriter:
         catalogUri = icebergSettings.catalogUri,
         additionalProperties = icebergSettings.additionalProperties, 
         s3CatalogFileIO = icebergSettings.s3CatalogFileIO,
-        locationOverride = icebergSettings.locationOverride,
+        locationOverride = icebergSettings.stagingLocation,
```

Comment:
i disagree with this rename - staging location is a ZIO term, not an Iceberg catalog term

Reply:
Discussed by voice: this field is part of the `StreamContext` provided by the `Arcane.Operator` and created from the custom resource. For the user (who works with this custom resource) it is not `locationOverride`, it's a 'location of staging table' and `stagingLocation` is better name then `locationOverride` in this file.